### DISCUSSION TITLE
Command line arguments?

### DISCUSSION THREAD

**Discussion by daytonbrown** [philosowaffle/peloton-to-garmin#613] [Ideas]
Running the windows exe, is there anyway to start it with an argument or 2 to just have it download the last x number of workouts and then quit?  I'm downloading them into fit files that I'm loading to both garmin, and then extracting the info into sql server.  Thanks to all the devs for this product.  Amazing.

**Comment by geoffbon** [comment]
There is a command line but I think it is limited to the same settings available in the config.  Some details [here](https://philosowaffle.github.io/peloton-to-garmin/latest/configuration/).
I haven't seen a setting for auto-closing, but it doesn't sound hard to add.

**Comment by daytonbrown** [comment]
I could be missing it, but it doesn't appear to have an arg for grab last xx, download and quit.
